### PR TITLE
Feat/project update

### PR DESCRIPTION
**Pull request by masaditya**
https://streamable.com/aoo2k1

### CODE REVIEW

File: src/modules/LoggedIn/Project/ProjectForm.tsx [hunk 1]
```diff
@@ -1,16 +1,27 @@
...
-  const toggleModal = React.useCallback(() => {
-    setVisible(prev => !prev);
-  }, []);
+
+  React.useEffect(() => {
+    setNewData({
+      name: props.project ? props.project.name : "",
+      description: props.project ? props.project.description : "",
+    });
+  }, [props]);
```
Comment (mnindrazaka) [inline src/modules/LoggedIn/Project/ProjectForm.tsx]: harusnya dependency nya cukup `props.project` aja

```suggestion
  }, [props.project]);
```

File: src/modules/LoggedIn/Project/ProjectForm.tsx [hunk 2]
```diff
@@ -1,16 +1,27 @@
 import React, { ChangeEvent } from "react";
 import { Button, Modal } from "antd";
-import { useProjectSync, ProjectRequest } from "./useProjectSync";
-const ProjectForm = () => {
+
+import { useProjectSync, ProjectRequest, Project } from "./useProjectSync";
+
+type FormProps = {
+  visible: boolean;
+  toggleModal: () => void;
```
Comment (mnindrazaka) [inline src/modules/LoggedIn/Project/ProjectForm.tsx]: nama props biasain pake prefix on, misal `onToggleModal`

```suggestion
  onToggleModal: () => void;
```

**Review by mnindrazaka — CHANGES_REQUESTED** [review]

**Comment by masaditya** [comment]
Maaf mas, masih conflict sama `master` branch. minta tolong di solve ya hehe 
*takut sama Formik wkwkw

**Review by mnindrazaka — APPROVED** [review]
LGTM

**Pull request merged**